ThemingCodelab/app/src/main/java/com/codelab/theming/ui/finish/theme/Theme.kt

```diff
@@ -36,7 +36,7 @@ private val LightColors = lightColors(
 private val DarkColors = darkColors(
     primary = Red300,
     primaryVariant = Red700,
-    onPrimary = Color.Black,
+    onPrimary = Color.White,
```

Comment:
Are you sure this needs to be changed?

Reply:
Let's not merge yet. Discussing this with @nickbutcher